### DISCUSSION TITLE
How  do I run or deploy a  wamp server

### DISCUSSION THREAD

**Discussion by leo-corleone** [crossbario/autobahn-java#550] [Q&A]
I can use docker to deploy or run a wamp server container ?  whether the docker hub has wamp image.

**Comment by pavelvic** [comment]
Hi! 
autobahn-java is a wamp **client** library. For starting wamp server you can use crossbario - https://hub.docker.com/r/crossbario/crossbar

**Comment by leo-corleone** [comment]
> Hi! autobahn-java is a wamp **client** library. For starting wamp server you can use crossbario - https://hub.docker.com/r/crossbario/crossbar

Thanks for your answer. But I can't browse docker official site. Do you have other any solution?
